lib/content-services/document-list/components/document-list.component.ts

```diff
@@ -590,6 +590,10 @@ export class DocumentListComponent implements OnInit, OnChanges, OnDestroy, Afte
     }
 
     private loadTrashcan(merge: boolean = false): void {
+        this.folderNode = null;
```

Comment:
instead to duplicate this reset can we move it in a method?

Reply:
Sure, I will do that. Thanks, Eugenio.